tests/smoke/services_test.go

```diff
@@ -29,8 +30,28 @@ var _ = Describe("Services", func() {
 
 	Describe("cf create-service", func() {
 		It("creates a managed service", func() {
-			session := helpers.Cf("create-service", "sample-service", "sample", "-b", brokerName, uuid.NewString())
+			session := helpers.Cf("create-service", "sample-service", "sample", uuid.NewString(), "-b", brokerName)
 			Expect(session).To(Exit(0))
 		})
 	})
+
+	Describe("cf services", func() {
+		var serviceName string
+
+		BeforeEach(func() {
+			serviceName = uuid.NewString()
+			session := helpers.Cf("create-service", "sample-service", "sample", serviceName, "-b", brokerName)
+			Expect(session).To(Exit(0))
+		})
+
+		It("lists services", func() {
+			session := helpers.Cf("services")
+			Expect(session).To(Exit(0))
+
+			lines, _ := it.Collect2(it.LinesString(session.Out))
```

Comment:
In `go-functional` `v2.1.0` I've added the `MustCollect` consumer for cases where you know errors will not occur or where panicing is OK (such as in tests). This could be written more clearly as:

```go
lines := it.MustCollect(it.LinesString(session.Out))
```